src/validation/rules/recommended/endpointsRecommendedBodyParameterPostfix.rule.ts

```diff
@@ -0,0 +1,49 @@
+import { ASTNodeKind } from './../../../language/ast';
+import { AnySpecError } from '../../../error';
+import { ASTVisitor } from '../../../visitor';
+import { ValidationContext } from '../../validationContext';
+
+const prefixMap = {
+  POST: 'CreateRequestBody',
+  PATCH: 'UpdateRequestBody',
+};
+
+/**
+ *
+ *
+ * good ✅
+ *
+ * POST /endpoint RequestCreateRequestBody
+ * PATCH /endpoint2 RequestUpdateRequestBody
+ *
+ * bad ❌
+ *
+ * POST /endpoint Request
+ * PATCH /endpoint2 Request
+ *
+ */
+export function EndpointsRecommendedQueryPostfix(context: ValidationContext): ASTVisitor {
+  return {
+    EndpointTypeDefinition(node) {
+      const keys = Object.keys(prefixMap);
+      if (keys.includes(node.verb.name.value)) {
+        const verb = node.verb.name.value as keyof typeof prefixMap;
```

Comment:
what about using predicate that we discuss today, to get rid of hacky typecast

() is Something